cost-analyzer/values.yaml

```diff
@@ -327,8 +327,8 @@ grafana:
       label: grafana_dashboard
     datasources:
       enabled: true
-      defaultDatasourceEnabled: true
-      dataSourceName: Prometheus
```

Comment:
how was this line alone not breaking grafana given that we no longer create this datasource?

Reply:
Setting default to false is all that's needed now. dataSourceName would only be for BYO grafana going forward.